games-simulation/flightgear/flightgear-2018.2.2.recipe

```diff
@@ -0,0 +1,108 @@
+SUMMARY="An Amazing 3D Flight Simulator"
+DESCRIPTION="FlightGear creates a sophisticated flight simulator framework \
+for use in research or academic environments, pilot training, as an industry \
+engineering tool, for DIY-ers to pursue their favorite interesting flight \
+simulation idea, and last but certainly not least as a fun, realistic, and \
+challenging desktop flight simulator. We are developing a sophisticated, open \
+simulation framework that can be expanded and improved upon by anyone interested \
+in contributing."
+HOMEPAGE="https://home.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/flighgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="924bcaec4bead47d01638f1033c0ca95c4a3fff65bd43d9c5b171d354b4b28f6"
+ADDITIONAL_FILES="flightgear.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	flightgear$secondaryArchSuffix
+	cmd:fgcom
+	cmd:fgelev
+	cmd:fgfs
+	cmd:fgjs
+	cmd:fgpanel
+	cmd:fgtraffic
+	cmd:fgviewer
+	cmd:GPSsmooth
+	cmd:JSBSim
+	cmd:js_demo
+	cmd:metar
+	cmd:MIDGsmooth
+	cmd:UGsmooth
+	cmd:yasim
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:harfbuzz$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+  lib:libgl$secondaryArchSuffix
+  lib:libglu$secondaryArchSuffix
+  lib:libopenal$secondaryArchSuffix
+  lib:libosg$secondaryArchSuffix
+	lib:libplib$secondaryArchSuffix
+  lib:libqt5$secondaryArchSuffix
+	lib:libsimgear$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:harfbuzz$secondaryArchSuffix
+	devel:libboost_system$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libglu$secondaryArchSuffix
+  devel:libopenal$secondaryArchSuffix
+  devel:libosg$secondaryArchSuffix
+	devel:libplib$secondaryArchSuffix
+  devel:libqt5$secondaryArchSuffix
+	devel:libsimgear$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:make
+	cmd:cmake >= 3.0
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. \
+		-DFG_DATA_DIR:STRING="/boot/system/data/flightgear" \
```

Comment:
```diff
-		-DFG_DATA_DIR:STRING="/boot/system/data/flightgear" \
+		-DFG_DATA_DIR:STRING="`finddir B_SYSTEM_DATA_DIRECTORY /flightgear" \
```

BTW, the indentation is incorrect in a few lines of `REQUIRES`, `BUILD_REQUIRES` and `INSTALL`: you have spaces instead of a tab.